lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2019/ashrae_90_1_2019.AirLoopHVAC.rb

```diff
@@ -317,16 +317,60 @@ def air_loop_hvac_supply_air_temperature_reset_required?(air_loop_hvac, climate_
       return is_sat_reset_required
     end
 
+    # check if design outside air is less than 10,000cfm (5000L/s) 90.1 2019 6.5.3.5 Exception 1 and 2
+    design_oa_m3s = air_loop_hvac.sizingSystem.designOutdoorAirFlowRate.to_f
+    if design_oa_m3s < 0.0001
+      design_oa_m3s = air_loop_hvac.sizingSystem.autosizedDesignOutdoorAirFlowRate.to_f
+    end
+    design_oa_cfm = OpenStudio.convert(design_oa_m3s, 'm^3/s', 'cfm').get
+
+    # check if there is erv 90.1 2019 Exceptions to 6.5.3.5 Exception 3
+    has_erv = air_loop_hvac_energy_recovery?(air_loop_hvac)
+    design_sa_m3s = air_loop_hvac.designSupplyAirFlowRate.to_f
```

Comment:
same issue of the designSupplyAirFlowRate being an optional double.

Reply:
Modified. I use an existing method from Standards.AirLoopHVAC.rb to get this value.